README.md

```diff
@@ -72,19 +69,117 @@ require('calendar')
 ```
 app/javascript/calendar/index.js // gets loaded by require('calendar')
 app/javascript/calendar/components/grid.jsx
+app/javascript/calendar/styles/grid.sass
 app/javascript/calendar/models/month.js
 ```
 
-But it could also look a million other ways. The only convention that Webpacker enforces is the
-one where entry points are automatically configured by the files in `app/javascript/packs`.
+```erb
+<%# app/views/layout/application.html.erb %>
+<%= javascript_pack_tag 'calendar' %>
+<%= stylesheet_pack_tag 'calendar' %>
+```
+
+But it could also look a million other ways.
+
+## Advanced Configuration
+
+By default, webpacker offers simple conventions for where the webpack configs, javascript app files and compiled webpack bundles will go in your rails app,
+but all these options are configurable from `config/webpacker.yml`
+that comes with webpacker.
+
+```yml
+default: &default
+  paths:
+    src_path: app/javascript
+    config_path: config/webpack
+    node_modules_path: node_modules
+    dist_dir: packs
+    dist_path: public/packs
+
+  dev_server:
+    enabled: true
+    host: localhost
+    port: 8080
+    compress: true
+
+production:
+  <<: *default
+
+development:
+  <<: *default
+
+test:
+  <<: *default
+```
```

Comment:
It wouldn't actually, but I added just to follow the convention in other files (database.yml) and in case someone gets fancy to change per/env basis. You suggested in another comment to split this in 2 files - perhaps just do this without env and in two files?